Sources/JExtractSwiftLib/FFM/FFMSwift2JavaGenerator.swift

```diff
@@ -65,6 +63,9 @@ package class FFMSwift2JavaGenerator: Swift2JavaGenerator {
         return String(filePathPart.replacing(".swift", with: "+SwiftJava.swift"))
       })
       self.expectedOutputSwiftFiles.insert("\(translator.swiftModuleName)Module+SwiftJava.swift")
+
+      // FIXME: Can we avoid this?
+      self.expectedOutputSwiftFiles.insert("Data+SwiftJava.swift")
```

Comment:
Yeah, this is unavoidable in today's SwiftPM.

Can we call the file `Foundation_Data+SwiftJava.swift` maybe? just to lessen the chance of clashing with some user defined Data type?

FYI ongoing sadness with build plugins @bnbarham heh